flink-cdc-connect/flink-cdc-source-connectors/flink-cdc-base/src/test/java/org/apache/flink/cdc/connectors/base/JdbcConnectionPoolTest.java

```diff
@@ -26,72 +26,76 @@
-        MysqlPooledDataSourceFactory mysqlPooledDataSourceFactory =
-                new MysqlPooledDataSourceFactory();
         JdbcConnectionPools mockInstance =
                 JdbcConnectionPools.getInstance(mockConnectionPoolFactory);
-        JdbcConnectionPools mysqlInstance =
-                JdbcConnectionPools.getInstance(mysqlPooledDataSourceFactory);
-        MySqlSourceConfig mySqlSourceConfig =
-                getMockMySqlSourceConfig(HOSTNAME, PORT, USER_NAME, PASSWORD, DATABASE, TABLE);
 
-        Assert.assertEquals(
-                mockInstance.getJdbcUrl(
-                        mySqlSourceConfig, mockConnectionPoolFactory.getClass().getName()),
-                mockConnectionPoolFactory.getJdbcUrl(mySqlSourceConfig));
-        Assert.assertEquals(
-                mysqlInstance.getJdbcUrl(
-                        mySqlSourceConfig, mysqlPooledDataSourceFactory.getClass().getName()),
-                mysqlPooledDataSourceFactory.getJdbcUrl(mySqlSourceConfig));
-        Assert.assertNotEquals(
-                mysqlInstance.getJdbcUrl(
-                        mySqlSourceConfig, mysqlPooledDataSourceFactory.getClass().getName()),
-                mockConnectionPoolFactory.getJdbcUrl(mySqlSourceConfig));
+        assertThat(
+                        mockInstance.getJdbcUrl(
+                                mySqlSourceConfig, mockConnectionPoolFactory.getClass().getName()))
+                .isEqualTo(mockConnectionPoolFactory.getJdbcUrl(mySqlSourceConfig));
+        assertThat(
+                        mysqlInstance.getJdbcUrl(
+                                mySqlSourceConfig,
+                                mysqlPooledDataSourceFactory.getClass().getName()))
+                .isEqualTo(mysqlPooledDataSourceFactory.getJdbcUrl(mySqlSourceConfig));
+        assertThat(
+                        mysqlInstance.getJdbcUrl(
+                                mySqlSourceConfig,
+                                mysqlPooledDataSourceFactory.getClass().getName()))
+                .isNotEqualTo(mockConnectionPoolFactory.getJdbcUrl(mySqlSourceConfig));
     }
 
     @Test
-    public void testNoDataSourcePoolFactoryIdentifier() {
-        MysqlPooledDataSourceFactory mysqlPooledDataSourceFactory =
-                new MysqlPooledDataSourceFactory();
-        JdbcConnectionPools mysqlInstance =
-                JdbcConnectionPools.getInstance(mysqlPooledDataSourceFactory);
-        MySqlSourceConfig mySqlSourceConfig =
-                getMockMySqlSourceConfig(HOSTNAME, PORT, USER_NAME, PASSWORD, DATABASE, TABLE);
+    void testNoDataSourcePoolFactoryIdentifier() {
         ConnectionPoolId poolId =
                 new ConnectionPoolId(
                         HOSTNAME,
                         PORT,
                         USER_NAME,
                         DATABASE,
                         MockConnectionPoolFactory.class.getName());
-        Assert.assertThrows(
+        assertThrowsExactly(
```

Comment:
Wondering if we can use `assertThatThrownBy#isExactlyInstanceOf` to replace `assertThrowsExactly`, and replace JUnit assertions with AssertJ completely?

Reply:
Thanks @yuxiqian! Good catch! 

Indeed I was planning to migrate totally to assertj where possible, I will update this and check the rest of files 👍